.github/CODEOWNERS

```
@@ -88,3 +88,6 @@ xpti/ @tovinkere @andykaylor
 xptifw/ @tovinkere  @andykaylor
 
 llvm/tools/sycl-post-link/ @kbobrovs @AlexeySachkov
+
+clang/tools/clang-offload-bundler/ @kbobrovs @sndmitriev
+clang/tools/clang-offload-wrapper/ @kbobrovs @sndmitriev
```

```suggestion
clang/tools/clang-offload-wrapper/ @sndmitriev @kbobrovs
```
as @sndmitriev is the author of the tool

---

Sure, I didn't know that.